PACKAGE.md

```diff
@@ -106,7 +106,9 @@ To checkout in which codestreams the package is currently maintaned, run:
 osc -A https://api.suse.de maintained SUSEConnect
 ```
 
-For a more detailed view which target codestreams are in which state, check out: [Codestream overview](https://maintenance.suse.de/maintained/?package=SUSEConnect)
```

Comment:
Had a quick look at the source code of that page... I think you can replace `package` with `q`:

https://maintenance.suse.de/maintained/?q=suseconnect